app/views/pages/developer-guide/forms/form-rendering.liquid

```diff
@@ -92,14 +92,14 @@ Using the [`form tag`](/api-reference/liquid/platformos-tags#form) is the easies
   <input type="hidden" name="slug3">
   <input type="hidden" name="resource_id" value="new">
   <input type="hidden" name="parent_resource_id" value="example_model">
-  <input type="hidden" name="parent_resource_class" value="CustomModelType">
+  <input type="hidden" name="parent_resource_class" value="CustomRecordType">
```

Comment:
same here - CustomModelType, this is auto-generated input

Reply:
Edited, thank you!